examples/examples-gpu/nyc-taxi/rf-rapids.ipynb

```diff
@@ -4,47 +4,98 @@
    "cell_type": "markdown",
```

Comment:
<p>"OR need a refresher"</p><p> </p><p>Good notes on the <code>n_streams</code> stuff. Do we not care about that or want to mention it on the multiple machine cluster? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/saturncloud/examples/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
thanks! I decided not to mention it on the multi-node, multi-GPU example because I expect that `cuml` does some stuff with `n_streams` based on the number and size of Dask workers in your cluster. I expected that figuring out whether or not that's true would take an hour or two of research, and didn't think that was the best use of my time right now. If you disagree I'm happy to investigate!

Reply:
Nah, that sounds fine- I imagined there was a good reason!